app/src/main/res/layout/bottom_sheet_nearby.xml

```diff
@@ -4,13 +4,33 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:orientation="vertical"
+    android:gravity="bottom"
     app:behavior_hideable="true"
     android:visibility="visible"
     app:layout_behavior="@string/bottom_sheet_behavior"
-    android:background="@android:color/white">
+    android:background="@android:color/transparent">
+
+    <RelativeLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:orientation="vertical"
+      android:background="@android:color/white">
+
+    <TextView
+      android:id="@+id/no_results_message"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:paddingVertical="50dp"
```

Comment:
setting both paddingVertical and paddingHorizontal to the same value can be simplified to just setting `padding`